system/lib/jsmath.c

```diff
@@ -33,21 +33,3 @@ CALL_JS_1_TRIPLE(sqrt)
 
 CALL_JS_2_TRIPLE(atan2)
 CALL_JS_2_TRIPLE(pow)
-
-#define CALL_JS_1_IMPL(cname, type, impl) \
-  EM_JS(type, JS_##cname, (type x), impl); \
-  type cname(type x) { return JS_##cname(x); }
-
-#define CALL_JS_1_IMPL_TRIPLE(cname, impl) \
-  CALL_JS_1_IMPL(cname, double, impl) \
-  CALL_JS_1_IMPL(cname##f, float, impl)
-
-CALL_JS_1_IMPL_TRIPLE(round, {
```

Comment:
What round will be used instead? musl's does not look efficient,

https://github.com/emscripten-core/emscripten/blob/main/system/lib/libc/musl/src/math/round.c

Reply:
So maybe we just need to update musl's round to use the advice given in https://github.com/emscripten-core/emscripten/blob/1092ec30a3fb1d46b1782ff1b4db5094d3d06ae5/system/include/emscripten/em_math.h#L31?   According that comment we should be able to do better than calling out to JS, right?

Reply:
Oh wait, `fround` is not `round`.